layout-management-webapps/src/main/webapp/skin/less/siteNavigation.less

```diff
@@ -5,6 +5,26 @@
   display: flex;
 }
 #siteNavigationAddNodeDrawer {
+  .iconExternal-stream {
+    .fa();
+    .fa-user-lock();
+    font-size: 28px;
+  }
+  .iconStream {
+    .fa();
+    .fa-stream();
+    font-size: 28px;
+  }
+  .iconSpaces {
+    .fa();
+    .fa-people-arrows();
+    font-size: 28px;
+  }
+  .iconPeople {
+    .fa();
+    .fa-users();
+    font-size: 28px;
```

Comment:
we should not add specific css classes for DW icons, the node icon should be retrieved directly from the rest response